src/app/content/utils.ts

```diff
@@ -26,18 +27,19 @@ export interface ContentPageRefencesType {
   pageId: string;
 }
 
-export function getContentPageReferences(htmlContent: string) {
-  const matches: ContentPageRefencesType[] = (htmlContent.match(/.\/([a-z0-9-]+(@[\d.]+)?):([a-z0-9-]+.xhtml)/g) || [])
+export function getContentPageReferences(book: ArchiveBook, page: ArchivePage) {
+  // tslint:disable-next-line: max-line-length
```

Comment:
you should reformat the code to be shorter, rather than disabling the linter everywhere